ethers-core/src/types/chain.rs

```diff
@@ -47,6 +47,7 @@ pub type ParseChainError = TryFromPrimitiveError<Chain>;
 #[strum(serialize_all = "kebab-case")]
 #[repr(u64)]
 pub enum Chain {
+    #[strum(serialize = "ethlive")]
```

Comment:
this will change replace "mainnet" whith ethlive.

imo mainnet is more meaningful.

we could add this as alias, but dunno how with strum tbh.

Reply:
yeah it's weird, I learnt this not too long ago: you can have only one "to_string" attr, which is the main one. Then aliases use serialize = ...
I'll open a pr to add that to the comments above